ExpensesTracker/Client/Pages/MonthlyExps.razor

```diff
@@ -95,4 +112,66 @@
         await MonthlyExpService.DeleteExpense(id);
         NavigationManager.NavigateTo("/monthlyexp");
     }
+
+    private void OnInputFileChange(InputFileChangeEventArgs e)
+    {
+        selectedFiles = e.GetMultipleFiles();
+        message = "File selected";
```

Comment:
Not sure if, message is needed. There seems to be some build in feature near choose file button that shows chosen files. Either disable that or remove this I think.

Reply:
The message is there to not only inform user about selected and uploaded files, but also to tell about errors from wrong data input/format.